src/Workspaces/VisualBasic/Portable/Utilities/SpeculationAnalyzer.vb

```diff
@@ -89,6 +89,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Utilities
         Protected Overrides Sub ValidateSpeculativeSemanticModel(speculativeSemanticModel As SemanticModel, nodeToSpeculate As SyntaxNode)
             Debug.Assert(speculativeSemanticModel IsNot Nothing OrElse
                         TypeOf nodeToSpeculate Is ExpressionSyntax OrElse
+                        TypeOf nodeToSpeculate Is ExpressionStatementSyntax OrElse
```

Comment:
The problem that this addresses is that a speculative semantic model for a top level statement was not implemented in VB.
I have a patch that addresses this issue but I will send it in another PR.